package.json

```diff
@@ -4,11 +4,11 @@
   "description": "Lazyload your Component, Image or anything matters the performance.",
   "main": "lib/index.js",
   "scripts": {
-    "test": "./node_modules/karma/bin/karma start test/karma.conf.js",
-    "demo:watch": "./node_modules/.bin/webpack-dev-server --inline --config webpack.config.js --content-base examples --port 8721",
-    "demo:build": "NODE_ENV=production ./node_modules/.bin/webpack",
-    "build": "./node_modules/.bin/babel src/ --out-dir lib/ --stage 0 --loose all",
-    "lint": "./node_modules/.bin/eslint -c .eslintrc src/"
+    "test": "karma start test/karma.conf.js",
```

Comment:
npm scripts have `node_modules/.bin` on the path by default; removing the [path separators improves portability](https://gist.github.com/domenic/2790533#being-windows-developer-friendly)